### ISSUE TITLE
Investigate use of JSON for config files

### ISSUE THREAD

**Issue by robberwick** [hackhitchin/piwars3#11]
Currently, we're using INI files in order to persist configuration parameters in a file, which is quite verbose when serialising and deserialising. If we switched to using JSON, then we would be able to directly save and load a single python configuration dictionary by using `json.dumps(conf_obj)` and `json.loads(file_content)`.

**Comment by worlds6440** [comment]
I agree, INI file can be a little verbose in code for reading/writing. I will look into JSON files in a bit when I have more challenges written.

**Comment by robberwick** [comment]
INI files are perfectly fine in the short term, so there's no real gain in switching to JSON in preference to writing challenges. It's certainly not vital for the competition. I'm happy to look at this as a nice to have feature if you assign it to me.
